vm/compiler/src/program/instruction/operation/macros.rs

```diff
@@ -710,14 +710,36 @@ mod tests {
 
                     // Check the operation on randomly-sampled values.
                     for i in 0..num_iterations {
+                        macro_rules! sample_value {
```

Comment:
this macro should be defined outside of this loop

Reply:
The macro uses `i` in the loop's scope.
I can move it outside and have the macro take `i` as an argument, it just didn't feel as clean to me.
Happy to go with whichever you feel is better.

Reply:
nah, it's ok; I usually declare macros outside of loops, but since this doesn't result in them expanded any more than inside of them, there's no issue there